### PR TITLE
Add changesets for DAM scoping (PR #976)

### PR DESCRIPTION
**Pull request by thomasdax98**
Add changesets for DAM scoping (PR #976)

### CODE REVIEW

File: .changeset/ten-numbers-sip.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Add scoping to the DAM
+
+The DAM scoping can be enabled optionally. You can still use the DAM without scoping.
+
+To enable DAM scoping, you must
+
+- In the API: Pass a `Scope` class to the `DamModule`
+- In the admin app: Add a `DamConfigProvider` with `scopeParts` (e.g. `<DamConfigProvider value={{ scopeParts: ["domain"] }}>`)
+
+You can access the current DAM scope in the admin app using the `useDamScope()` hook.
```
Comment (thomasdax98) [inline .changeset/ten-numbers-sip.md:15]: @johnnyomair Maybe I missed something. Feel free to extend this

File: .changeset/ten-numbers-sip.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+Add scoping to the DAM
+
+The DAM scoping can be enabled optionally. You can still use the DAM without scoping.
+
+To enable DAM scoping, you must
+
+- In the API: Pass a `Scope` class to the `DamModule`
+- In the admin app: Add a `DamConfigProvider` with `scopeParts` (e.g. `<DamConfigProvider value={{ scopeParts: ["domain"] }}>`)
```
Comment (johnnyomair) [inline .changeset/ten-numbers-sip.md:13]:
```suggestion
- In the admin app: Set `scopeParts` in the `DamConfigProvider` (e.g. `<DamConfigProvider value={{ scopeParts: ["domain"] }}>`)
```

The DamConfigProvider was already present before

**Review by johnnyomair — COMMENTED** [review]
Thanks! Totally missed that 🙃

**Comment by johnnyomair** [comment]
@thomasdax98 I can't set you as reviewer since you created the PR... @nsams could you please review?

**Review by nsams — APPROVED** [review]

**Pull request merged**